pkg/3scale/amp/template/adapters/images.go

```diff
@@ -15,39 +14,37 @@ func NewImagesAdapter() Adapter {
 }
 
 func (i *ImagesAdapter) Parameters() []templatev1.Parameter {
-	imageProvider := product.CurrentImageProvider()
-
 	return []templatev1.Parameter{
 		templatev1.Parameter{
 			Name:     "AMP_BACKEND_IMAGE",
 			Required: true,
-			Value:    imageProvider.GetBackendImage(),
+			Value:    component.ApisonatorImageURL(),
```

Comment:
The same applies here. References to Apisonator should also be renamed to Backend (envvars, methods, ...)